include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -746,7 +746,7 @@ class __future : private std::tuple<_Args...>
     }
 
     template <typename _T, std::size_t _NResults>
-    constexpr auto
+    constexpr _T
```

Comment:
... I know this is not new code in this PR, but why is this `constexpr`?
It doesnt seem like `__wait_and_get_value` for `__result_and_scratch_storage` can ever be constexpr.  Can we just remove that?

Reply:
Fixed